src/Button.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Button.css'
+export default function Button(props) {
+   const {
+    children,
+    className,
+    disabled=false,
+    color='',
+    onClick,
+    variant='',
+    height='auto',
+    width='auto',
+    size='16px',
+    onMouseEnter,
+    onMouseLeave,
+    img,
+    src
+   } = props;
+  const buttonClass = `${className} ${variant?variant:""}`
```

Comment:
can add default values in props to these or cover the edge case if these are not defined